knit/env.py

```diff
@@ -0,0 +1,188 @@
+from __future__ import absolute_import, division, print_function
+
+import os
+import sys
+import shutil
+import requests
+import logging
+import zipfile
+from subprocess import Popen, PIPE
+
+from .exceptions import CondaException
+
+mini_file = "Miniconda-latest.sh"
+
+linux_miniconda_url = "https://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh"
+osx_miniconda_url = "https://repo.continuum.io/miniconda/Miniconda-latest-MacOSX-x86_64.sh"
+win_miniconda_url = "https://repo.continuum.io/miniconda/Miniconda-latest-Windows-x86_64.exe"
+
+logger = logging.getLogger(__name__)
+
+class CondaCreator(object):
+    """
+    Create Conda Env
+    """
+
+    def __init__(self, conda_root=None):
```

Comment:
I would be surprised if you could do this in a fairly general way.  I think it would possibly be desirable but nothing I'm concerned with at the moment.  But I do a lot of the development on OSX